2022-01-26 22:35:07,823 INFO    MainThread:18452 [wandb_setup.py:_flush():71] setting env: {}
2022-01-26 22:35:07,823 INFO    MainThread:18452 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-26 22:35:07,824 INFO    MainThread:18452 [wandb_init.py:_log_setup():357] Logging user logs to /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220126_223507-1zml0lax/logs/debug.log
2022-01-26 22:35:07,824 INFO    MainThread:18452 [wandb_init.py:_log_setup():358] Logging internal logs to /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220126_223507-1zml0lax/logs/debug-internal.log
2022-01-26 22:35:07,824 INFO    MainThread:18452 [wandb_init.py:init():390] calling init triggers
2022-01-26 22:35:07,824 INFO    MainThread:18452 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {'width': 25, 'height': 25, 'num_agents': 1, 'max_num_cities': 2, 'seed': 1, 'grid_mode': False, 'max_rails_between_cities': 2, 'max_rails_in_city': 4, 'malfunction_rate': 0, 'min_duration': 20, 'max_duration': 50, 'observation_builder': 'GraphObsForRailEnv', 'predictor': 'ShortestPathPredictorForRailEnv', 'view_semiwidth': 7, 'view_height': 30, 'offset': 25, 'observation_depth': 3, 'num_episodes': 200, 'start_epoch': 0, 'max_steps': 300, 'eps': 1, 'debug_print': False, 'load_memory': False, 'evaluation_episodes': 3, 'render': False, 'evaluation_interval': 10, 'save_model_interval': 10, 'start_lr_decay': 150, 'eps_decay': 0.992, 'learning_rate': 0.03, 'learning_rate_decay': 1.0, 'learning_rate_decay_policy': 1.0, 'run_title': 'no-multi-no-action', 'wandb_project_name': 'flatland-original', 'model_path': 'test_results/', 'model_name': '', 'gat_layers': 3, 'flow': 'source_to_target', 'resume_weights': True, 'batch_norm': False, 'dropout_rate': 0.8, 'attention_heads': 4, 'batch_size': 128, 'use_stop_action': False, 'done_reward': 0, 'deadlock_reward': -1000, 'reward_scaling': 0.1}
2022-01-26 22:35:07,824 INFO    MainThread:18452 [wandb_init.py:init():435] starting backend
2022-01-26 22:35:07,824 INFO    MainThread:18452 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-26 22:35:07,836 INFO    MainThread:18452 [backend.py:ensure_launched():199] starting backend process...
2022-01-26 22:35:07,845 INFO    MainThread:18452 [backend.py:ensure_launched():204] started backend process with pid: 18758
2022-01-26 22:35:07,846 INFO    MainThread:18452 [wandb_init.py:init():444] backend started and connected
2022-01-26 22:35:07,848 INFO    MainThread:18452 [wandb_init.py:init():503] updated telemetry
2022-01-26 22:35:07,870 INFO    MainThread:18452 [wandb_init.py:init():533] communicating current version
2022-01-26 22:35:08,814 INFO    MainThread:18452 [wandb_init.py:init():538] got version response 
2022-01-26 22:35:08,814 INFO    MainThread:18452 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2022-01-26 22:35:10,084 WARNING MainThread:18452 [wandb_init.py:init():891] interrupted
Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 870, in init
    run = wi.init()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 549, in init
    ret = backend.interface.communicate_run(run, timeout=30)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 205, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/interface/interface_queue.py", line 334, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/interface/interface_queue.py", line 232, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/interface/router.py", line 32, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
